inginious/frontend/plugins/register_students/static/email_course_registration_template.html

```diff
@@ -0,0 +1,30 @@
+<html>
+    <head></head>
+    <body>
+        <h2>$:_("Welcome on {course_name}")</h2>
+        
+        <h4>$:_("Hello {username}")</h4>
+        <br><br>
+        $:_("You have been registered for the course: <strong>{course_name}</strong> at Universidad Nacional de Colombia.")
+        <br><br>
+        $:_("If you have already activated your account, you can access to UNCode to start the course.")
+        <br><br>
+
+        <i>$:_("How to proceed?")</i><br>
+        <ol>
+            <li>$:_("When you logged in, you could see the main page with the courses that you are enrolled")</li>
+            <li>$:_("Click on <strong>{course_name}</strong> option on the course list or click this link:")
+                <ul><li>$:_("{course_link}")</li></ul>
```

Comment:
Leave this variable without the messages translation syntax. Also make sure to update de i18n files when change this and the above messages.